src/WSEvents.cpp

```diff
@@ -756,7 +758,7 @@ void WSEvents::OnExit() {
 }
 
 /**
- * Emit every 2 seconds.
+ * Emitted every 2 seconds when stream is active.
```

Comment:
It is emitted every 2 seconds after being manually turn on by calling `SetHeartbeat`